pkg/antctl/raw/check/installation/command.go

```diff
@@ -99,16 +102,25 @@ func Run(o *options) error {
 	if err := testContext.setup(ctx); err != nil {
 		return err
 	}
+	var numSuccess, numFailure, numSkipped int
 	for name, test := range testsRegistry {
 		testContext.Header("Running test: %s", name)
-		if err := test.Run(ctx, testContext); err != nil {
-			testContext.Header("Test %s failed: %s", name, err)
+		if err, skipped := test.Run(ctx, testContext); skipped {
+			testContext.Warning("Test %s was skipped: %v", name, err)
```

Comment:
Not sure if I missed some codes, do we provide a way to allow user to skip some tests? Looks like the `skipped` is always false. Maybe we can add this later when there is a way to skip test?